docs/usage/2-manifest.mdx

```diff
@@ -678,9 +678,13 @@ description: "Recommented settings including warning flags to help you catch som
 case: ".essential",
 description: "A minimal set of settings to make the project compile without any additional settings for example `PRODUCT_NAME` or `TARGETED_DEVICE_FAMILY`."
 },
+{
+case: ".none",
+description: "Tuist won't generate any build settings for the target or project."
+},
 ]}/>
 
-<Message info title="Essential Settings" description="The `.essential` option can be used in the event all warnings are defined in an `xcconfig` file to prevent Tuist from overriding them at the project or target level."/>
+<Message info title="Essential Settings" description="The .essential option can be used in the event all warnings are defined in an `xcconfig` file to prevent Tuist from overriding them at the project or target level."/>
 
```

Comment:
Worth adding a warning style comment for the `.none` case highlighting Tusit is delegating the responsibility of settings to the user and the project may fail to compile! :)

Reply:
How does this look?

> Projects may fail to compile if some essential settings are missing. Use `.none` only if you are specifying all the necessary build settings manually or via xcconfig files.

Reply:
Something like this @kwridan ?

![image](https://user-images.githubusercontent.com/663605/59136246-1af54900-8937-11e9-959e-83762b447689.png)